### DISCUSSION TITLE
💡【方法论】我的效率提升方法论 - 通用思维篇 ① 收件箱

### DISCUSSION THREAD

**Discussion by imageslr** [imageslr/blog#94] [Announcements]
https://imageslr.com/2021/efficiency-01.html 

 前言

**Comment by ArgentByte** [comment]
请问博主，你的域名无法访问是怎么解决的，我也有类似的问题

**Comment by imageslr** [comment]
> 请问博主，你的域名无法访问是怎么解决的，我也有类似的问题

@Aymetic 你是在 netlify 上部署的吗？具体现象是什么？

**Comment by imageslr** [comment]
@Aymetic 看一下这个帖子，我是把腾讯云的 dns 记录重置了一下。
https://answers.netlify.com/t/cant-access-my-site-via-https-in-china-mainland-without-vpn/40599

**Comment by jxau-hujin** [comment]
楼主大大，可以推荐几本非技术类书籍吗？

**Comment by imageslr** [comment]
> 楼主大大，可以推荐几本非技术类书籍吗？

我读的书也不是很多。请问是哪些方面的书？我可以分享一下自己看过的资源。

**Comment by jxau-hujin** [comment]
> @imageslr
> > 楼主大大，可以推荐几本非技术类书籍吗？
> 
> 我读的书也不是很多。请问是哪些方面的书？我可以分享一下自己看过的资源。

比如关于效率提升方面的方法论这种

**Comment by imageslr** [comment]
@jxau-hujin 关于效率提升，我读过《Zen to Done》《Get Things Done》，以及[褪墨・时间管理](https://www.mifengtd.cn/)的许多文章，也尝试过 Things、OmniFocus、滴答清单、奇妙清单等时间管理工具。平时也会在少数派、Notion 上看看他人的效率提升方法。

我觉得大部分效率提升的思维都是通用的，更多的还是要在实践中养成。建议是你每次专注一至两个习惯，先用一段时间来摸索、调整为适合自己的流程，形成肌肉记忆，然后再去学习新的习惯。否则如果只看书、书又很厚的话，容易抓不住重点。

推荐看看《Zen to Done》。

**Comment by imageslr** [comment]
### 提升个人信息管理效率：思路与工具

在字节学习上发现一个有关效率提升的课程，听下来感觉很不错，也有很多一致的想法。作者是字节的同学，其本质工作和信息效率强相关。之前在少数派卖过信息效率的课程，有几千人购买。这次分享也是上述课程的整理、浓缩与视频版。

课程内容：任务管理 GTD、信息管理、看板、个人知识库。

我的视角，重点内容：

1. GTD。看起来执行流程图复杂，实际只有收集、整理、执行、回顾 四个部分。
    - 收集：随时用身边任意工具快速记录事情。清空大脑，同时避免其他事情打断当前工作。
    - 整理：有空的时候，把收件箱的条目整理，比如移动至对应分类、设定提醒时间等。收件箱需要清空。执行：选择本周要做、今天要做、下一步要做的事。这个最好和OKR结合，这些事应该是KR的分解。
    - 回顾：定期重新整理所有清单，调整任务优先级，重新分配双月计划、周计划等。  
    
   我一直在实行GTD。使用的工具是滴答清单。

2. 信息管理。作者提出，飞书和邮箱也可以用GTD的思维管理。
    - 飞书的未读消息等价于一个任务，可以点击每个消息的“✓”把它标记为已完成。同理，邮件也适合定期清理。
    - 不要让飞书消息只有已读和未读 (小红点) 两个状态，几千条红点未读带来的提醒会让你不堪重负。善用置顶、允许通知、免打扰、会话盒子和标记，多层级地管理自己的会话。  
    
   我日常也是这样使用飞书。每天只会收到100-300条消息，可以轻松清空未读，重要消息可以秒读。

3. 看板。这个可以和GTD结合。GTD主要是分类管理任务，每个任务有未开始、开始、完成状态。但GTD一般是以列表形式展示所有任务，看板的展示形式更直观，能一眼看到不同状态的任务，也适合拖拖拽拽。滴答清单有这个功能，但我平时用的不多。
4. 知识库。有意识地打造个人知识库，整理一批原子笔记，积累并输出系统的笔记。这里我想补充的是，笔记也可以用GTD的思想管理：随时记录有用的信息，定期整理，在个人知识库分类存档。个人知识库要选支持搜索的 + 支持 markdown 的。

总结：信息时代，效率就是生产力。提升效率不一定能够显著提升工作产出，但能明显节省重复劳动的时间，差距是日积月累产生的。学习方法和工具有阵痛期，但跳出舒适区一段时间后，会豁然开朗，整体效率直线上升。**像打造产品一样，打造个人的生产力系统。**
